libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -1630,10 +1642,10 @@ void GCS_MAVLINK::send_accelcal_vehicle_position(uint32_t position)
   motors. That can be dangerous when a preflight reboot is done with
   the pilot close to the aircraft and can also damage the aircraft
  */
-MAV_RESULT GCS_MAVLINK::handle_preflight_reboot(const mavlink_command_long_t &packet, bool disable_overrides)
+MAV_RESULT GCS_MAVLINK::handle_preflight_reboot(const mavlink_command_long_t &packet)
 {
     if (is_equal(packet.param1,1.0f) || is_equal(packet.param1,3.0f)) {
```

Comment:
it would be much nicer if this if was inverted and we returned early. While at it, fix the coding style to have spaces

Reply:
Pushing a patch on top which tidies it all up.